tests/src/PatternAssertion/InPageNavigationAssert.php

```diff
@@ -23,6 +23,16 @@ protected function getAssertions(string $variant): array {
       'links' => [
         [$this, 'assertList'],
       ],
+      'id' => [
+        [$this, 'assertElementAttribute'],
+        '.bcl-inpage-navigation',
+        'id',
+      ],
+      'dropdown_id' => [
+        [$this, 'assertElementAttribute'],
+        '.dropdown-toggle',
+        'id',
+      ],
```

Comment:
you can just do this in assertBaseElements

Reply:
We actually can't, as the value changes and that method cannot access any values passed.